src/block/carousel/edit.js

```diff
@@ -262,9 +261,13 @@ const Edit = props => {
 		sliderRef.current.parentElement.addEventListener( 'mouseenter', onMouseEnter )
 		sliderRef.current.parentElement.addEventListener( 'mouseleave', onMouseLeave )
 		return () => {
-			clearInterval( interval )
-			sliderRef.current.parentElement.removeEventListener( 'mouseenter', onMouseEnter )
-			sliderRef.current.parentElement.removeEventListener( 'mouseleave', onMouseLeave )
+			try {
+				clearInterval( interval )
+				sliderRef.current.parentElement.removeEventListener( 'mouseenter', onMouseEnter )
+				sliderRef.current.parentElement.removeEventListener( 'mouseleave', onMouseLeave )
+			  } catch ( err ) {
+
+			  }
```

Comment:
I think the error here is from `sliderRef.current` being undefined? If it is, then I think optional chaining would be better here: 

```js
sliderRef.current?.parentElement.removeEventListener( 'mouseenter', onMouseEnter )
```